### PR TITLE
DOC Fix broken syntax

### PR DESCRIPTION
**Pull request by GuySartorelli**
The notice and the nested code block are rendering incorrectly:

image

## Issue
- https://github.com/silverstripe/developer-docs/issues/363

### CODE REVIEW

File: en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md [hunk 1]
```diff
@@ -204,15 +204,16 @@ A couple things to note here:
...
-MyProject\Models\Product:
-  fields:
-    onSale:
-      type: Boolean
-```
+    ```yaml
+    MyProject\Models\Product:
+    fields:
+        onSale:
+        type: Boolean
```
Comment (sabina-talipova) [inline en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md]: Is it correct? `onSale` property doesn't have any value?

**Review by sabina-talipova — CHANGES_REQUESTED** [review]

File: en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md (hunk 1 above)
Comment (ChloeSartorelli) [reply en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md]: Oops, I missed an extra level of indentation there. Will fix it.

File: en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md [hunk 2]
```diff
@@ -239,9 +240,9 @@ elemental: # An arbitrary key to define what these directives are doing
   load:
     inheritanceLoader:
       include:
-          - DNADesign\Elemental\Models\BaseElement
+        - DNADesign\Elemental\Models\BaseElement
       exclude:
-          - MyApp\Elements\MySecretElement
+        - MyApp\Elements\MySecretElement
```
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md:245]: Should have always been 2 spaces of indentation.

File: en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md (hunk 1 above)
Comment (ChloeSartorelli) [reply en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md]: Done

**Review by sabina-talipova — APPROVED** [review]
Looks good to me.

**Pull request merged**